controllers/apply/awards-for-all/form.test.js

```diff
@@ -865,6 +880,26 @@ describe('Contacts', () => {
         });
     });
 
+    test('contact address required by default', () => {
```

Comment:
This test probably isn't _strictly_ required as it's covered by the amended empty form test but worth having an an explicit check.